website/docs/r/cluster.html.markdown

```diff
@@ -200,20 +200,49 @@ resource "mongodbatlas_cluster" "cluster-test" {
 }
 ```
 ### Example - Return a Connection String
-AWS 
+AWS Private Endpoint
 ```hcl
 output "plstring" {
     value = lookup(mongodbatlas_cluster.cluster-test.connection_strings[0].private_endpoint[0].srv_connection_string, aws_vpc_endpoint.ptfe_service.id)
 }
 //Example return string: plstring = mongodb+srv://cluster-atlas-pl-0.za3fb.mongodb.net
 ```
-Azure
+Azure Private Endpoint
 ```hcl
 output "plstring" {
     value = lookup(mongodbatlas_cluster.cluster-test.connection_strings[0].private_endpoint[0].srv_connection_string, azurerm_private_endpoint.test.id)
 }
 //Example return string: plstring = mongodb+srv://cluster-atlas-pl-0.za3fb.mongodb.net
 ```
+Standard
+```hcl
+output "standard_srv" {
```

Comment:
[nit] `output "standard_srv"` -> `output "standard_srv"` because here we are setting `mongodb://..` connection string and not `mongodb+srv://...`

Reply:
thanks for the eye, anuj. just fixed it